content/en/docs/components/training/reference/architecture.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Architecture"
+description = "The Training Operator Architecture"
+weight = 10
++++
+
+{{% stable-status %}}
+
+## What is the Training Operator Architecture?
+
+The original design was drafted in April 2021 and is [available here for reference](https://docs.google.com/document/d/1x1JPDQfDMIbnoQRftDH1IzGU0qvHGSU4W6Jl4rJLPhI/).
+The goal was to provide a unified Kubernetes operator that supports multiple
+machine learning/deep learning frameworks. This was done by having a "Frontend"
+operator that decomposed the job into different configurable Kubernetes
+components (e.g., Role, PodTemplate, Fault-Tolerance, etc.). The "Backend"
+operator watches all Role Customer Resources and manages pod performance.
+
+The benefits of this approach were:
+1. Shared testing and release infrastructure
+2. Unlocked production manifests and metadata support
```

Comment:
What do you mean by that ?

Reply:
This is taken directly from the [Advantage section](https://docs.google.com/document/d/1x1JPDQfDMIbnoQRftDH1IzGU0qvHGSU4W6Jl4rJLPhI/edit#heading=h.p4xrerz4bt7h) of the original documentation which said
>Unlock more production grade features like manifests/metadata supports.

I can rephrase this. 👍